src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve --host 0.0.0.0",
-    "build": "ng build",
+    "build": "ng build && node ./copyright.build.js",
```

Comment:
After running `npm build`, I still see the placeholder instead of the current year:

![screenshot from 2019-01-04 12-09-21](https://user-images.githubusercontent.com/14297426/50688274-9ebfa800-101c-11e9-9d87-3bee29d5a568.png)

To fix this, the build should be `"node ./copyright.build.js && ng build"` instead of `"ng build && node ./copyright.build.js"`.